Content.Client/Audio/Jukebox/JukeboxMenu.xaml.cs

```diff
@@ -86,33 +71,43 @@ private void PlaybackSliderKeyUp(Slider args)
     /// </summary>
     public void Populate(IEnumerable<JukeboxPrototype> jukeboxProtos)
     {
-        MusicList.Clear();
-
         foreach (var entry in jukeboxProtos)
         {
-            MusicList.AddItem(entry.Name, metadata: entry.ID);
+            // MusicList.AddItem(entry.Name, metadata: entry.ID);
+            var songControl = new JukeboxEntry(entry) {EntryType = JukeboxEntry.Type.List};
+            songControl.SetOnPressedPlay((song, _, args) =>
+            {
+                if (song == null)
+                    return;
+                OnSongSelected?.Invoke(song.ID);
+                OnPlayPressed?.Invoke(true);
+            });
+
+            songControl.SetOnPressedQueue((song, args) =>
+            {
+                if (song == null)
+                    return;
+
+
+                OnSongQueueAdd?.Invoke(song.ID);
+            });
+
+            MusicList.AddChild(songControl);
         }
```

Comment:
Looks like the added changes break prototype reloading now. Same with the music queue not handling it so if the jukebox prototypes reload it just exceptions.